pkg/utils/controller/run.go

```diff
@@ -13,7 +13,7 @@ import (
 	"k8s.io/client-go/util/workqueue"
 )
 
-type reconcileFunc func(string, string, string) error
+type reconcileFunc func(logr.Logger, string, string, string) error
```

Comment:
Can we add keys for these three strings? It's not clear to me what they represent.